### PR TITLE
Support armv7hl arch in tizen

### PR DESCRIPTION
**Pull request by chunseoklee**
- armv7hl in cmake config files
- armv7hl in spec file

Change-Id: If9ce0012703948b2b76e17c753e29af94939b9fd
Signed-off-by: Chunseok Lee <chunseok.lee@samsung.com>

### CODE REVIEW

**Comment by hseok-oh** [comment]
@nnfw-bot test tizen-gbs-builder

File: infra/nnfw/cmake/options/options_armv7hl-tizen.cmake [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#
+# armv7hl tizen cmake options
+#
+option(BUILD_ARMCOMPUTE "Build ARM Compute from the downloaded source" OFF)
+option(BUILD_TENSORFLOW_LITE "Build TensorFlow Lite from the downloaded source" OFF)
+option(DOWNLOAD_NEON2SSE "Download NEON2SSE library source" OFF)
+
+option(BUILD_LOGGING "Build logging runtime" OFF)
+option(GENERATE_RUNTIME_NNAPI_TESTS "Generate NNAPI operation gtest" OFF)
+option(ENVVAR_ONERT_CONFIG "Use environment variable for onert configuration" OFF)
```
Comment (hseok-oh) [inline infra/nnfw/cmake/options/options_armv7hl-tizen.cmake:10]: @chunseoklee Don't we allow `gpu_cl` backend?

Comment (chunseoklee) [reply infra/nnfw/cmake/options/options_armv7hl-tizen.cmake:10]: I have committed patch for gpu_cl.

**Review by hseok-oh — APPROVED** [review]
LGTM

**Pull request merged**